recipes/fltk/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "fltk" %}
+{% set version = "1.3.4" %}
+{% set longversion = "1.3.4-2" %}
+{% set md5 = "69128be72365cbedd1d380739f0164e3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ longversion }}-source.tar.gz
+  url: http://fltk.org/pub/{{ name }}/{{ version }}/{{ name }}-{{ longversion }}-source.tar.bz2
+  md5: {{ md5 }}
+  patches:
+    - macros.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - zlib
+    - jpeg
+    - libpng
```

Comment:
Change these to `libpng >=1.6.32,<1.6.35`, `zlib 1.2.11`, `jpeg 9*` and the same in run?